data/committees.js

```diff
@@ -497,14 +517,14 @@ export default [
 				platform: 'website',
 				link: 'https://cloud.uclaacm.com/',
 			},
-			{
-				platform: 'instagram',
-				link: 'https://www.instagram.com/acmcloud/',
-			},
 			{
 				platform: 'discord',
 				link: 'https://discord.gg/smE3pk5DsJ',
```

Comment:
looks like this one expired - i've asked cloud for a new one that doesn't expire

Reply:
https://discord.com/invite/s2VGxKqTYA